src/backend_model.cc

```diff
@@ -1022,7 +1022,12 @@ TRITONBACKEND_RequestIsCancelled(
     TRITONBACKEND_Request* request, bool* is_cancelled)
 {
   InferenceRequest* tr = reinterpret_cast<InferenceRequest*>(request);
-  *is_cancelled = tr->IsCancelled();
+
+  auto status = tr->IsCancelled(is_cancelled);
+  if (!status.IsOk()) {
+    return TRITONSERVER_ErrorNew(
```

Comment:
Can we use [macro](https://github.com/triton-inference-server/core/blob/a23786c25b36f07d2cfd510821682c117b50a99e/src/status.h#L88C9-L94) here?
```cpp
RETURN_TRITONSERVER_ERROR_IF_ERROR(tr->IsCancelled(is_cancelled));
```